pkg/yttlibrary/url.go

```diff
@@ -221,7 +226,7 @@ func (uv *URLValue) AsStarlarkValue() starlark.Value {
 }
 
 func (uv *URLValue) ConversionHint() string {
-	return "URLValue: cannot coerce to a string; use .string()"
+	return "URLValue: does not automatically encode; use .string()"
```

Comment:
instead of using semi-colon, can we use either `--` or, similar to some other messages `(hint: use .string())`

Reply:
Yeah, elsewhere we're using the `(hint: %s)` format... in an effort to be consistent (and I find the potential corrective action easier to find that way).

Fixed.